### PR TITLE
Rich text featured snippets

### PR DESCRIPTION
**Pull request by cea2aj**
Support rich text featured snippets

Note: This PR leverages the current develop version of the search ui. Before actual release, this should be updated to point to v1.9.

This PR
- Adds a fieldName: 'snippet' parameter to analytics events on the direct answer card
- Tweaks the RTF styling to match the updated mocks
- Updates the direct answer card styling in general

This PR also bumps the default font size of regular direct answers so that it matches this new font size of featured snippet direct answers. This was approved by product.

J=SLAP-1384
TEST=manual

Add data to the knowledge graph and test various featured snippet queries. Also test on the yext help site which has rtf featured snippets enabled.

### CODE REVIEW

**Review by yen-tt — COMMENTED** [review]
Should we add a snapshot in photographer ( in _captureDirectAnswers) with a query for rich text snippet? Also, are we including a max/fixed height for the card for when it's too long?

**Comment by cea2aj** [comment]
> Should we add a snapshot in photographer ( in _captureDirectAnswers) with a query for rich text snippet? Also, are we including a max/fixed height for the card for when it's too long?

Yes, we should! I can do that as soon as I get the version of the SDK with the necessary core version working

**Comment by cea2aj** [comment]
> Should we add a snapshot in photographer ( in _captureDirectAnswers) with a query for rich text snippet? Also, are we including a max/fixed height for the card for when it's too long?

Regarding max/fixed height, currently no, we are going to display the whole answer by default. If someone would like, we will provide instructions for setting a max height with a scroll bar. In the future we may develop a more advanced way of limiting the height.

File: static/scss/answers/common/mixins.scss [hunk 1]
```diff
@@ -135,7 +135,7 @@
     margin-inline-end: var(--yxt-base-spacing);
   }
 
-  h1 
+  h1, .js-yxt-rtfValue > *:first-child // The first item should also get h1 styling
```
Comment (cea2aj) [inline static/scss/answers/common/mixins.scss:138]: oops, I need to make sure this only applies to rich text featured snippets. Thanks percy!

File: hbshelpers/sdkAssetUrl.js [hunk 2]
```diff
@@ -17,7 +17,10 @@ module.exports = function sdkAssetUrl(branch, locale, assetName) {
   let parsedBranch;
   if (isReleasedBranch) {
     parsedBranch = `v${branch}`;
-  } else {
+  } else if (branch === 'develop') {
+    parsedBranch = 'canary/latest';
+  }
+  else {
```
Comment (oshi97) [inline hbshelpers/sdkAssetUrl.js:23]: nit can we have the else one line above

File: test-site/config/global_config.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "sdkVersion": "1.8", // The version of the Answers SDK to use
+  "sdkVersion": "develop", // The version of the Answers SDK to use
```
Comment (oshi97) [inline test-site/config/global_config.json:2]: just checking this is intentional!

File: global_config.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "sdkVersion": "1.8", // The version of the Answers SDK to use
+  "sdkVersion": "develop", // The version of the Answers SDK to use
```
Comment (oshi97) [inline global_config.json:2]: is this one intentional too?

File: directanswercards/documentsearch-standard/component.js [hunk 5]
```diff
@@ -13,14 +13,21 @@ class documentsearch_standardComponent extends BaseDirectAnswerCard['documentsea
    */
   dataForRender(type, answer, relatedItem, snippet) {
     const relatedItemData = relatedItem.data || {};
+    let snippetValue = "";
```
Comment (oshi97) [inline directanswercards/documentsearch-standard/component.js:16]: nit can we use single quotes

File: directanswercards/documentsearch-standard/component.js [hunk 6]
```diff
@@ -13,14 +13,21 @@ class documentsearch_standardComponent extends BaseDirectAnswerCard['documentsea
...
+    if (answer.fieldType == "rich_text" && snippet) {
```
Comment (oshi97) [inline directanswercards/documentsearch-standard/component.js:17]: should this be triple equals?

**Review by oshi97 — COMMENTED** [review]
besides nits lgtm!

File: test-site/config/global_config.json (hunk 3 above)
Comment (cea2aj) [reply test-site/config/global_config.json:2]: yes, we need to be on develop for this to work

File: hbshelpers/sdkAssetUrl.js (hunk 2 above)
Comment (cea2aj) [reply hbshelpers/sdkAssetUrl.js:23]: my bad

File: global_config.json (hunk 4 above)
Comment (cea2aj) [reply global_config.json:2]: yes, we'll switch it to 1.9 when that's released (or we can switch it to the release branch if we cut those)

File: directanswercards/documentsearch-standard/component.js (hunk 6 above)
Comment (cea2aj) [reply directanswercards/documentsearch-standard/component.js:17]: this is what happens when I rely too closely on product's spec

File: global_config.json (hunk 4 above)
Comment (cea2aj) [reply global_config.json]: updated to point to release/v1.9

**Review by oshi97 — APPROVED** [review]
lgtm!

**Pull request merged**